File: programmers/java/lev0/k의 개수.java
public class k의 개수 {
    
}
class Solution {
    public int solution(int i, int j, int k) {
        String str = "";
        for(int a = i; a <= j; a++) {
            str += a+"";
        }

        return str.length() - str.replace(k+"", "").length();
    }
}


class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;

        for (int num = i; num <= j; num++){
            int tmp = num;
            while (tmp != 0){
                if (tmp % 10 == k)
                    answer++;
                tmp /= 10;
            }
        }
        return answer;
    }
}

File: programmers/js/lev0/k의 개수.js
function solution(i, j, k) {
  let answer = 0;
  while (i <= j) {
    answer += String(i)
      .split("")
      .filter((el) => el === String(k)).length;
    i += 1;
  }
  return answer;
}

function solution(i, j, k) {
  let a = "";
  for (i; i <= j; i++) {
    a += i;
  }
  return a.split(k).length - 1;
}
